data/transition-sites/dclg_localdirect.yml

```diff
@@ -0,0 +1,9 @@
+---
+site: dclg_localdirect
+whitehall_slug: department-for-communities-and-local-government
+homepage: https://www.gov.uk/government/organisations/department-for-communities-and-local-government
```

Comment:
...and there's even [`css: department-for-communities-and-local-government`](https://github.com/alphagov/transition-config/blob/76ffb59fc56d6be64d83a0b4283648c76be3b5d1/data/transition-sites/dclg.yml#L12) too!